integration_tests/e2eReferDisabled/refer.cy.ts

```diff
@@ -86,4 +86,23 @@ context('Refer', () => {
     const notFoundPage = Page.verifyOnPage(NotFoundPage)
     notFoundPage.shouldContain404H2()
   })
+
+  it("Doesn't show the check answers page for a referral", () => {
+    cy.signIn()
+
+    const course = courseFactory.build()
+    const courseOffering = courseOfferingFactory.build()
+    const prison = prisonFactory.build({ prisonId: courseOffering.organisationId })
+    const referral = referralFactory.build()
+
+    cy.task('stubCourseByOffering', { course, courseOfferingId: courseOffering.id })
+    cy.task('stubCourseOffering', { courseId: course.id, courseOffering })
+    cy.task('stubPrison', prison)
```

Comment:
I think the prisoner and referral will need stubbing here as well since those endpoints would be hit by the relevant controller action - we want to be certain we aren't getting a 404 because those resources aren't found in the API vs the page not existing in the UI